src/um/winbase.rs

```diff
@@ -999,6 +1001,16 @@ pub const STARTF_TITLEISLINKNAME: DWORD = 0x00000800;
 pub const STARTF_TITLEISAPPID: DWORD = 0x00001000;
 pub const STARTF_PREVENTPINNING: DWORD = 0x00002000;
 pub const STARTF_UNTRUSTEDSOURCE: DWORD = 0x00008000;
+STRUCT!{struct STARTUPINFOEXA {
+    StartupInfo: STARTUPINFOA,
+    lpAttributeList: PPROC_THREAD_ATTRIBUTE_LIST,
```

Comment:
This should be `LPPROC_THREAD_ATTRIBUTE_LIST` instead of `PPROC_THREAD_ATTRIBUTE_LIST`.

Reply:
Done.